frontend/app/component/form/differences/DifferencesForm.tsx

```diff
@@ -141,6 +141,7 @@ export function DifferencesForm() {
             id="more_ballots_count"
             title="Stembiljetten méér geteld"
             errorsAndWarnings={isSaved ? errorsAndWarnings : undefined}
+            warningsAccepted={getIgnoreWarnings()}
```

Comment:
For the future, maybe this is a good idea?
```
const defaultProps = { warningsAccepted: getIgnoreWarnings(), errorsAndWarnings: isSaved ? errorsAndWarnings : undefined, format};

<InputGridRow {...defaultProps />
```

Reply:
Good suggestion, implemented in c7faface31a7468c5ae56be5091677ca00e44261